big_tests/tests/service_domain_db_SUITE.erl

```diff
@@ -543,6 +546,37 @@ db_restarts_properly(_) ->
         end,
     mongoose_helper:wait_until(F, true, #{time_left => timer:seconds(15)}).
 
+db_loads_domains_after_node_joins_cluster(Config) ->
```

Comment:
This is not exactly the scenario that I had in mind, but a much simpler one: put a bunch of domains one one node, then add an empty second node to cluster with the first, then we expect this second node to serve the same domains the first does.
How does this scenario you propose proves my scenario is also valid?

Reply:
> an empty second node

It is _never_ empty, if it is running. 
It gets domains from DB.

Reply:
That is explicitly not true. From my tests, it only loads the static toml config, nothing else. Try this (using macros from `dynamic_domains_SUITE`
```erlang
    insert_domains([mim()], ?DOMAINS),
    Config1 = cluster_nodes(?CLUSTER_NODES, Config),
    R1 = rpc(mim(), ets, tab2list, [mongoose_domain_core]),
    ct:pal("Value ~p~n", [R1]),
    R2 = rpc(mim2(), ets, tab2list, [mongoose_domain_core]),
    ct:pal("Value ~p~n", [R2]),
    ok.
```
And you'll see they report different values.

Reply:
This code:
- does not start service_domain_db on both of nodes.
- does not start service_domain_db after clustering.
- You can check that service is running by calling `rpc(mim(), mongoose_service, is_loaded, [service_domain_db])`
- it uses mongoose_domain_core:insert/3 to insert domains. This function does not touches database and should be only used internally or for testing.
- use mongoose_domain_api:insert_domain instead.

Reply:
Manually running this branch against that test still returns different ets tables. Am I wrong to expect both ets tables to be equal?